java/integTest/src/test/java/glide/TransactionTestUtilities.java

```diff
@@ -96,9 +97,10 @@ public static BaseTransaction<?> transactionTest(BaseTransaction<?> baseTransact
 
     public static Object[] transactionTestResult() {
         return new Object[] {
-            OK,
-            value1,
-            null,
+            OK, // set(key1, value1)
+            value1, // get(key1)
+            null, // set(key2, value2, returnOldValue(true))
+            (long) value1.length(), // strlen(key2)
```

Comment:
@SanHalacogluImproving we should start adding these comments for the transaction list results.  The result value list is getting unreadable and this will make it easier to review.

Reply:
Agreed I think we should make a PR just to add them before adding a bunch of commands.

Reply:
if you decide on this approach, please do it in a different pr.
you can also look at node and python wrappers and see if that way is suitable for Java.